server.js

```diff
@@ -0,0 +1,8 @@
+const http = require('http');
```

Comment:
Please don't end lines with `;` if it's not strictly necessary. This also goes for subsequent lines.